basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/DBLTIService.java

```diff
@@ -475,7 +477,7 @@ public Object updateContentDao(Long key, Object newProps, String siteId,
             joinClause = joinClause + " LEFT JOIN SAKAI_SITE_PROPERTY ssp3 ON (lti_content.SITE_ID = ssp3.SITE_ID AND ssp3.name = '" + propertyKey + "')";
         }
         
-        final String[] fields = (String[])ArrayUtils.addAll(LTIService.CONTENT_MODEL, LTIService.CONTENT_EXTRA_FIELDS);
+        final String[] fields = (String[]) ArrayUtils.addAll(LTIService.CONTENT_MODEL, LTIService.CONTENT_EXTRA_FIELDS);
```

Comment:
I don't get how this was working before when there was no ArrayUtils import - or wasn't it?.  You're only adding a single space.